app/controllers/users_controller.rb

```diff
@@ -75,6 +75,39 @@ def destroy
 
   private
 
+    def user_params
+      if current_user && current_user.admin?
+        params.require(:user).permit!
```

Comment:
Does this say allow everything for admins?  I think you'd still want to restrict what params are permitted since you may have malicious admins!